src/components/ChatLog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const entryComponents = props.entries.map((entry) => {
+        return (
+            <ChatEntry
+                key={`${entry.sender}-${entry.timeStamp}`}
+                id={entry.id}
+                sender={entry.sender}
+                body={entry.body}
+                timeStamp={entry.timeStamp}
+                liked={entry.liked} 
+                updateLike={props.updateLike} />
+        );
+    });
+
+    return (
+        <ul className="chat-log">{entryComponents}</ul>
+    );
+}
+
+ChatLog.propTypes = {
+    entries: PropTypes.arrayOf(PropTypes.shape({
+        id: PropTypes.number,
+        sender: PropTypes.string.isRequired,
+        body: PropTypes.string.isRequired,
+        timeStamp: PropTypes.string.isRequired,
+        liked: PropTypes.bool
+    })),
+
+    updateLike: PropTypes.func,
```

Comment:
This should probably be `.isRequired` because having this callback is vital. However, that would likely break the test code we wrote, so it's fair that you didn't.